recipe/setup.py.patch

```diff
@@ -0,0 +1,59 @@
+diff -urN libsharp-1.0.0_orig/python/setup.py libsharp-1.0.0/python/setup.py
```

Comment:
Some version of this patch might be good to submit upstream.

Reply:
The upstream maintainer (Martin Reinecke) has a whole new version of libsharp out.  So I don't think this patch will live very long...  Will definitely try to upstream any fixes needed for libsharp 2.